app/scripts/modules/core/help/helpContents.js

```diff
@@ -527,4 +527,14 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'markdown.examples': 'Some examples of markdown syntax: <br/> *<em>emphasis</em>* <br/> **<b>strong</b>** <br/> [link text](http://url-goes-here)',
     'application.enableRestartRunningExecutions': 'When this option is enabled, users will be able to restart pipeline stages while a pipeline is still running. ' +
     'This behavior can have varying unexpected results and is <b>not recommended</b> to enable.',
+    'pipeline.config.webhook.payload': 'Json payload to be added to the webhook call.',
+    'pipeline.config.webhook.waitForCompletion': 'If not checked, we consider the stage succeeded if the webhook return an HTTP status code 2xx, otherwise it will be failed. If checked, it will poll a status url (defined below) to grasp the progress of the stage.',
+    'pipeline.config.webhook.useStatusUrlFromLocationHeaderTrue': 'Pick the status url from the Location header of the webhook\'s response call.',
+    'pipeline.config.webhook.useStatusUrlFromLocationHeaderFalse': 'Pick the status url from a json returned by the webhook\'s response call.',
+    'pipeline.config.webhook.statusUrlJsonPath': 'JsonPath where to find the status url in the webhook\'s response json. (i.e. <samp>$.buildInfo.url</samp>)',
+    'pipeline.config.webhook.statusJsonPath': 'JsonPath where to find the status information in the webhook\'s response json. (i.e. <samp>$.buildInfo.status</samp>)',
+    'pipeline.config.webhook.progressJsonPath': 'JsonPath where to find the progress information in the webhook\'s response json. (i.e. <samp>$.buildInfo.progress</samp>)',
```

Comment:
Progress mean a descriptive error message to be shown to the user, I rewrote it as: `JSON path to a descriptive message about the progress in the webhook\'s response JSON. (e.g. <samp>$.buildInfo.progress</samp>)` WDYT?